docker-compose.dev.yml

```diff
@@ -0,0 +1,15 @@
+version: "2"
+services:
+  app:
+    build:
+      context: .
+      dockerfile: build/Dockerfile.dev
+    image: go-api-template
+    volumes:
+      - ./internal:/go/src/ctco-dev/go-api-template/internal
+      - ./cmd:/go/src/ctco-dev/go-api-template/cmd
+    ports:
+      - 3006:3006
+    environment:
+      - JOKE_SERVICE_URL
```

Comment:
I can’t start the service without creating a dotenv file then. Why not just hardcore default values in the compose dev file and allow for overrides via dotenv?

Reply:
actually it works w/o `.env`, default value is provided for `envconfig` library. 
so that you can run the project event w/o docker

Reply:
Where does it pull the value from then?

Reply:
I put it here deliberately in code so that guys w/o docker could play with the project right away
https://github.com/ctco-dev/go-api-template/pull/19/files#diff-4f1556d13b82a883bbec405ba781f7d5R11

Reply:
Cool, thx, can we do the same for PORT?

Reply:
What about hard-coded port value in `docker-compose.yml` and hard-coded default value in source code and also potentially one that might be specified in `.env` file?
Not sure how to keep them in sync without added complexity to `docker-compose.yml` any thoughts?

Reply:
ok I see, let's think about it later